mappings/net/minecraft/util/math/MathUtils.mapping

```diff
@@ -50,17 +64,25 @@ CLASS none/or net/minecraft/util/math/MathUtils
 		ARG 0 d
 	METHOD c sqrt (F)F
 		ARG 0 f
+	METHOD c hsvToRgb (FFF)I
+		ARG 0 h
```

Comment:
Imo these should be expanded to the full names, along with other parameters in here: `hue`, `saturation`, `value`, and also `value` instead of `f`, `d`, `i` and such.